packages/conform-validator/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@hono/conform-validator",
+  "version": "0.0.0",
+  "description": "Validator middleware using Conform",
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
+  "types": "dist/esm/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "test": "vitest --run",
+    "build:cjs": "tsc -p tsconfig.cjs.json",
```

Comment:
Please use `tsup` like the `hello` middleware:

https://github.com/honojs/middleware/blob/main/packages/hello/package.json#L13

If it's a hassle for you, I can fix it.

Reply:
OK. I fixed it with reference to hello middleware.

commit: https://github.com/honojs/middleware/pull/666/commits/e1b58f86145e72fded5c1a4237a8bd01a230cb29